models/infra.js

```diff
@@ -0,0 +1,16 @@
+const subjectContent = new connector.model("subject content", subjectContentSchema);
+module.exports = {};
```

Comment:
can you remove this? this is probably from another file. please dont delete your branch, just remove this, commit it, and then push your work, on your branch.

Reply:
ok